code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -5,7 +5,7 @@
 	icon_locked = "fridge1"
 	icon_opened = "fridgeopen"
 	icon_off = "fridgebroken"
-	req_access = list(access_service)
```

Comment:
Wrong way to do it.
This list requires the user to have ALL of these accesses to open it.

I believe you want to change it to req_one_access which allows access if you have anything in the list